caravel/assets/spec/javascripts/sqllab/VisualizeModal_spec.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import VisualizeModal from '../../../javascripts/SqlLab/components/VisualizeModal';
+import { queries } from './common';
+import { Modal } from 'react-bootstrap';
+import { shallow } from 'enzyme';
+import { describe, it } from 'mocha';
+import { expect } from 'chai';
+
+
+describe('VisualizeModal', () => {
+  const mockedProps = {
+    show: true,
+    query: queries[0],
+  };
+  it('should just render', () => {
```

Comment:
for naming tests, i prefer to leave out the should part on each test, so these would become `it('renders')`, `it('renders with props')`. when the tests run they'll then read like:
```
VisualizeModal
- renders ✔
- renders with props ✔
- renders a Modal ✔
```